src/Error/Error.php

```diff
@@ -262,6 +278,19 @@ public function getPath(): ?array
         return $this->path;
     }
 
+    /**
+     * Returns an array describing the path from the root value to the field which produced this error.
+     * Only included for execution errors. This will never include aliases.
+     *
+     * @return array<int, int|string>|null
```

Comment:
I kept it `array<int, int|string>` but personally I would make this a `list<int|string>`.

Reply:
Let's change it to `list`, good idea!

Reply:
Changed it!